fair_mango/metrics/metrics.py

```diff
@@ -43,6 +63,116 @@ def __init__(
         self.label = label
 
     def __call__(self) -> tuple[Sequence[str], list[dict]]:
+        """Calculates selection rate for different sensitive groups.
+
+        Returns
+        -------
+        tuple[Sequence[str], list[dict]]
```

Comment:
improve your types

Reply:
It is now `tuple[Sequence[str], list[dict[str, np.ndarray]]]`